pvanalytics/quality/util.py

```diff
@@ -58,3 +58,37 @@ def check_limits(val, lower_bound=None, upper_bound=None,
         return ub_op(val, upper_bound)
     else:
         raise ValueError('must provide either upper or lower bound')
+
+
+def daily_maxmin(series, maximum, inclusive=False):
+    """Select only data on days where the daily minimum is less than minimum.
```

Comment:
```suggestion
    """Return True for data on days when the daily minimum is greater than (or equal to) daily_min.
```
Let's flip this around and return True for days when when min(series) > minimum